src/data/move-attrs/one-hit-ko-accuracy-attr.ts

```diff
@@ -15,7 +15,7 @@ export class OneHitKOAccuracyAttr extends VariableAccuracyAttr {
     if (user.level < target.level) {
       accuracy.value = 0;
     } else {
-      accuracy.value = Math.min(Math.max(30 + 100 * (1 - target.level / user.level), 0), 100);
+      accuracy.value = Math.min(30 + user.level - target.level, 100);
```

Comment:
```suggestion
      accuracy.value = Math.max(Math.min(move.accuracy + user.level - target.level, 100), 0);
```
Negative values cause the move to always hit, so this also needs a lower bound. This also doesn't assume a base accuracy of 30.

I'm also wondering if there's a way to reduce redundant code for Sheer Cold's attribute...

Reply:
This should never be in the negatives because this is in the `else` block so user.level will always be >= target.level